controller/cache/info.go

```diff
@@ -370,7 +412,7 @@ func populatePodInfo(un *unstructured.Unstructured, res *ResourceInfo) {
 	// and https://github.com/kubernetes/kubernetes/issues/90358#issuecomment-617859364
 	if pod.DeletionTimestamp != nil && pod.Status.Reason == "NodeLost" {
 		reason = "Unknown"
-	} else if pod.DeletionTimestamp != nil {
+	} else if pod.DeletionTimestamp != nil && !IsPodPhaseTerminal(podPhase) {
```

Comment:
why would podDeletionTimestamp not be nil, but IsPodPhaseTerminal be true?

Reply:
Thanks for your review!

I found this [issue](https://github.com/kubernetes/kubectl/issues/1595), and the dashboard PR could potentially reference this [change](https://github.com/kubernetes/kubernetes/pull/122038/files#diff-8083c7028e9b5553d36b3edd6d4a533c7a20e8ba883990de84b061d728572cd0R993).

So when a pod is in a Succeeded or Failed state, but has a finalizer and deletionTimestamp, based on this issue, we should prevent it from being displayed as "Terminating."

Reply:
makes sense, thank you for the clarification. maybe add a comment?

Reply:
added